systems/plants/RigidBodyManipulatorURDF.cpp

```diff
@@ -116,6 +116,28 @@ string resolveFilename(const string& filename, const map<string,string>& package
   return mesh_filename_s.getStr();
 }
 
+class WarningManager {
+  public:
+    void warnOnce(const string& id, const string& msg)
+    {
+      auto print_warning_iter = already_printed_warnings.find(id);
+      if (print_warning_iter == already_printed_warnings.end()) {
+        cout << msg << endl;
+        already_printed_warnings.insert(id);
+      }
+    }
+
+    void reset()
+    {
+      already_printed_warnings.clear();
+    }
+
+  private:
+    set<string> already_printed_warnings;
+};
+
+static WarningManager warning_manager;
```

Comment:
thanks!  isn't it better to do this once per robot (like in matlab), by having this as a member variable of RBM?